test/functional/feature_proxy.py

```diff
@@ -234,7 +234,13 @@ def networks_dict(d):
             return r
 
         self.log.info("Test RPC getnetworkinfo")
-        n0 = networks_dict(self.nodes[0].getnetworkinfo())
+        nodes_network_info = []
+        for node in self.nodes:
```

Comment:
I think adding documentation as follows would be helpful for future readers of this test.
```diff
         self.log.info("Test RPC getnetworkinfo")
         nodes_network_info = []
+
+        self.log.debug("Test that setting -proxy deactivates local address discovery (-discover=0)")
         for node in self.nodes:
             network_info = node.getnetworkinfo()
             assert_equal(network_info["localaddresses"], [])
```

Reply:
I agree, going to add it!

Reply:
done!